### DISCUSSION TITLE
When the Dragger is small, it cannot be activated.

### DISCUSSION THREAD

**Discussion by luohuiiqng** [openscenegraph/OpenSceneGraph#1313] [Q&A]
Hello! I'm currently trying to use the predefined node dragger in my project, and for the most part it works fine. However, when the Dragger was very small and close to the earth surface, I found that only EarthManipulator responded to the left mouse button click and drag event, while osgManipulator::Dragger did not respond. How can I modify them to make them work properly? Thank you!
![image](https://github.com/openscenegraph/OpenSceneGraph/assets/82392471/14e1bdaf-029f-4290-a100-9e21fdf7a1ae)

**Comment by luohuiiqng** [comment]
I found that the problem disappeared when the camera was not so close to the node.

**Comment by robertosfield** [comment]
With large coordinate systems it's important to place a MatrixTransform above subgraphs that have geometries in them to allow the subgraph to be setup with local coordinates while the the MatrixTransform places it in world coordinates.

**Reply by luohuiiqng** [reply]
Ok,Thank you!
